File: Python/modul5_kel46.py
# mengimport class userService dari file userService
from userService import userService

print("System Login Kelompok 46!\n")
# membaca input user dan menyimpan dalam variabel email
email = input("Email: ")
# membaca input user dan menyimpan dalam variabel password
password = input("Password: ")
# membuat constructor baru userService dengan isi parameter input email dan password 
# dengan variabel get_class
get_class = userService(email,password)
# memanggil function login pada class userService
get_class.login()

File: Java/modul5_kel46.java
import java.util.Scanner;


public class modul5_kel46 {
    public static void main(String[] args) {
        String email, password;
        
        Scanner input = new Scanner(System.in);
        System.out.print("Email : ");
        email = input.nextLine();
        System.out.print("Password : ");
        password = input.nextLine();
        
        userService satu = new userService( email, password );
        satu.login();
    }
}

        